packages/app/src/components/Root/LogoIcon.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import DevHubIcon from './logo/DevHubIcon.png';
+import DevHubIcon from './logo/BCID_V_RGB_rev.svg';
 
 export const LogoIcon = () => {
-  return <img width="64px" alt="DevHub Icon" src={DevHubIcon}/>;
+  return <img width="64px" height="auto" alt="DevHub Icon" src={DevHubIcon}/>;
```

Comment:
for accessibility purposes, a better alt text here might be something like "The Province of British Columbia logo with blue mountains and sun behind the mountains. The words British Columbia are below, with a decorative gold line that matches the sun colour beneath it."

@Shivanganii  please suggest something better! ^^